lib/tables.h

```diff
@@ -18,6 +18,9 @@ typedef int32_t population_id_t;
 typedef int32_t site_id_t;
 typedef int32_t mutation_id_t;
 typedef int32_t migration_id_t;
+typedef int32_t individual_id_t;
+typedef int32_t individual_sex_t;
+typedef double individual_age_t;
```

Comment:
I don't think we need a typedef for this. The IDs are typedef'd here in case we ever need to change to 64 bit table sizes. Then changing should be easy (well, easier). Times are measured in doubles elsewhere, so that's fine.